### DISCUSSION TITLE
Missing icons under Astroid menu options

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#651] [General]
Hello,

I think someone reported about this a few days ago, but I can't find the post anymore. But maybe I am wrong.

In the Astroid menu options (Select Icon) a lot of icons are not displayed for me.

I have now noticed this under Astroid 3.0.13 and also under Astroid 3.0.14.

My development environment is:

[Bearsampp](https://bearsampp.com/) WAMP, PHP 8.2.10, MariaDB 11.3.2 and Joomla 4.4.3.

I have also tried once with xampp 8.2.12, Joomla 4.4.3 and Astroid 3.0.14, but many icons are not displayed there either.

I don't know if this is a bug or if it has to be like this.

Translated with DeepL.com (free version)

![Astroid_Menu_Options](https://github.com/templaza/astroid-framework/assets/157874871/28c6f764-7c7f-465d-91b9-16e56a5f3dd6)

**Comment by sonvnn** [comment]
In the administrator, Astroid is using original fontawesome of Joomla library. But Joomla 4.4.3 is using Font Awesome Free 5.15.4. The latest Font Awesome is 6.5.2. You should update to Joomla 5. This issue will be resolve.
